controller/src/beerocks/master/tasks/optimal_path_task.cpp

```diff
@@ -1191,10 +1191,8 @@ void optimal_path_task::work()
                             << sta_mac << " steer from AP " << current_hostap_vap << " to AP "
                             << chosen_hostap;
             bool disassoc_imminent = true;
-            int disassoc_timer_ms  = 100;
-            steering_task_id =
-                son_actions::steer_sta(database, cmdu_tx, tasks, sta_mac, chosen_hostap,
-                                       disassoc_imminent, disassoc_timer_ms);
+            steering_task_id       = son_actions::steer_sta(database, cmdu_tx, tasks, sta_mac,
```

Comment:
I don't understand the commit message.
Also, did the signature of `steer_sta()` changed?
You replaced an `int` variable with a `bool` one...

Reply:
now that the "disassoc_timer" default is a valid number (200ms) no need to set it when calling steer_clint() since its set to default in client_steering_task() class constructor.